src/scrollable.directive.ts

```diff
@@ -99,7 +99,7 @@ export class NguiScrollableDirective {
     // detect the current environment
     let parentElStyle = window.getComputedStyle(parentEl);
     let scrollContainerEl = parentElStyle.overflow === 'auto' ?
-      parentEl : document.body;
+      parentEl : document.scrollingElement || document.documentElement;
```

Comment:
Interesting, I thought it is supported by all browesers. http://caniuse.com/#search=documentElement. Anyway, I will merge and release it since it does not harm any.